### PR TITLE
Add javascript

### PR DESCRIPTION
**Pull request by fuadmoin**
we added a pop-up window for both mobile and desktop versions.
when the user taps on each card their detail will be displayed on the pop-up window.
All the cards in the index .html are displayed dynamically using JavaScript.
There is now a link to the live demo and source code in the popup window.

### CODE REVIEW

**Review by t-yanick — COMMENTED** [review]
Hi @fuadmoin  , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) Linters set up

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -1,448 +1,380 @@
...
-            </div>
+          <section id="mobile-card-work1"></section>
+          <!--Desktop part-->
+          <section id="desktop-card-work1" class="card-collection"></section>
+          <section id="desktop-card-work2" class="card-collection"></section>
+          <div id="mobile-card-work2"></div>
         </div>
-    </div>
-    <div class="about-skill">
+      </div>
```
Comment (MeqdamAlqudah) [inline index.html:180]: - [ ] Please The popup should be responsive, it should fit mobile and desktop sizes properly look how the `X` button isn't appearing in this screen size.
image
look at this weird space beside the popup in screen size, fix it, please 
image

Comment (MeqdamAlqudah) [inline index.html:180]: - [Optional] Please it would be better to control the overflow for the popup, the user shouldn't scroll to find the   popup
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @fuadmoin , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

image

 ## Highlights
:+1: no linter errors 
:+1: Readme file

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nahnahdev — APPROVED** [review]
Hi @fuadmoin , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- [ ] Fixed all requested changes :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**